arangod/Aql/SortNode.cpp

```diff
@@ -123,8 +122,8 @@ bool SortNode::simplify(ExecutionPlan* plan) {
 }
 
 /// @brief returns all sort information
-SortInformation SortNode::getSortInformation(
-    ExecutionPlan* plan, arangodb::basics::StringBuffer* buffer) const {
+SortInformation SortNode::getSortInformation(ExecutionPlan* plan,
+                                             std::string& buffer) const {
```

Comment:
`std::string&` -> `std::string&&`?

Reply:
I simplified the API of this method even more. Turned out `plan` wasn't even used by the method. And it feels clumsy to fix in an std::string for temporary string buildup, so I removed that from the API and instead used a local std::string value inside the method.